rocketpy/motors/liquid_motor.py

```diff
@@ -264,7 +273,34 @@ def exhaust_velocity(self):
         mass flow rate. Therefore, this will vary with time if the mass flow
         rate varies with time.
         """
-        return self.thrust / (-1 * self.mass_flow_rate)
+        if not isinstance(self.thrust.source, np.ndarray):
+            return self.thrust / (-1 * self.mass_flow_rate)
+
+        times, thrusts = self.thrust.source[:, 0], self.thrust.source[:, 1]
+        mass_flow_rates = self.mass_flow_rate(times)
+
+        # Avoid division by zero by replacing 0 with NaN
+        mass_flow_rates_nonzero = np.where(
+            np.isclose(mass_flow_rates, 0), np.nan, mass_flow_rates
+        )
+        ext_vel = -thrusts / mass_flow_rates_nonzero
```

Comment:
Well, can you run `pytest -k "test_liquid_motor_info" and see the ZeroDivisionError?

If the mass_flow is zero at some point, how do you calculate the exhaust velocity? It should be undefined I think (nan).

I even tried to change the truediv_ of Function class to try/except divisions per zero, but it was no easy task.

Therefore, I don't wanna say that I love the approach here, but it is definitely necessary.

I don't find any nan_to_num in Function.

Reply:
Again, this is important to avoid divisionByZero errors. See the latest commit to get the new behavior: https://github.com/RocketPy-Team/RocketPy/pull/470/commits/d7e4dba4970debd89088ece6a6990a934a74b35f